pkg/apis/toolchain/v1alpha1/hostoperatorconfig_types.go

```diff
@@ -0,0 +1,89 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// HostOperatorConfigSpec contains all configuration parameters of the host operator
+// +k8s:openapi-gen=true
+type HostOperatorConfigSpec struct {
+	// Keeps parameters necessary for automatic approval
+	// +optional
+	AutomaticApproval AutomaticApproval `json:"automaticApproval,omitempty"`
+}
+
+// Defines all parameters necessary for automatic approval
+// +k8s:openapi-gen=true
+type AutomaticApproval struct {
+	// Defines if the automatic approval is activated or not
+	Activated bool `json:"activated"`
```

Comment:
Maybe enabled would be better?
```suggestion
	// Defines if the automatic approval is enabled or not
	Enabled bool `json:"enabled"`
```

Reply:
I was thinking about `Enabled` as well, but I chose `Activated` as a way to say if the automatic approval is active or not.
Which of those do you like more @xcoulon ?

Reply:
Active/inactive sounds more like something for status. Not for spec :)
But I'm fine with both options. Just felt that enabled is something usually used for enabling/disabling features. But we can keep "activated" if you prefer. It's clear what it means anyway and it's minor.

Reply:
You are right. I changed it in 2b29853